source/configuring-ember/debugging.md

```diff
@@ -34,13 +29,6 @@ ENV.APP.LOG_VIEW_LOOKUPS = true;
 Ember.keys(Ember.TEMPLATES)
 ```
 
-#### Handlebars Debugging Helpers
```

Comment:
They are covered in http://guides.emberjs.com/v1.13.0/templates/development-helpers/. I've tried to keep this page to be about configuration options as much as possible. (It could still use some cleanup, but I wanted to get it in the right place at least.)